test/extended/tests-extension/event_ttl.go

```diff
@@ -0,0 +1,514 @@
+package extended
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	operatorclientset "github.com/openshift/client-go/operator/clientset/versioned"
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/operatorclient"
+	test "github.com/openshift/cluster-kube-apiserver-operator/test/library"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+)
+
+var _ = g.Describe("[Jira:kube-apiserver][sig-api-machinery][FeatureGate:EventTTL] Event TTL Configuration", func() {
+	var (
+		kubeClient     *kubernetes.Clientset
+		configClient   *configclient.Clientset
+		operatorClient *operatorclientset.Clientset
+		ctx            context.Context
+	)
+
+	g.BeforeEach(func() {
+		ctx = context.TODO()
+		kubeConfig, err := test.NewClientConfigForTest()
+		o.Expect(err).NotTo(o.HaveOccurred(), "failed to get kube config")
+
+		kubeClient, err = kubernetes.NewForConfig(kubeConfig)
+		o.Expect(err).NotTo(o.HaveOccurred(), "failed to create kube client")
+
+		configClient, err = configclient.NewForConfig(kubeConfig)
+		o.Expect(err).NotTo(o.HaveOccurred(), "failed to create config client")
+
+		operatorClient, err = operatorclientset.NewForConfig(kubeConfig)
+		o.Expect(err).NotTo(o.HaveOccurred(), "failed to create operator client")
+	})
+
+	// Loop to create separate test cases for each TTL value (5m, 10m, 15m)
+	testValues := []int32{5, 10, 15}
+	for _, ttlMinutes := range testValues {
+		ttl := ttlMinutes
+
+		g.It(fmt.Sprintf("should configure and validate eventTTLMinutes=%dm [Timeout:90m][Disruptive][Slow][Suite:openshift/cluster-kube-apiserver-operator/conformance/serial]", ttl), func() {
```

Comment:
add parameter `ote.Informing()`, don't put it in blocking CI queue without enouth testing on different scenarios or envs.